src/xpk/commands/cluster.py

```diff
@@ -423,6 +425,23 @@ def run_gke_cluster_delete_command(args) -> int:
   Returns:
     0 if successful and 1 otherwise.
   """
+  if not args.force:
+    xpk_print('Get the name of the workloads in the cluster.')
+    args.filter_by_status = 'EVERYTHING'
+    return_code, return_value = get_workload_list(args)
+    if return_code != 0:
+      xpk_print(f'List Job request returned ERROR {return_code}')
+      return return_code
+
+    workloads = [x.split(' ')[0] for x in return_value.splitlines()][1:]
+    if workloads and not get_user_input(
+        f'Planning to delete {len(workloads)} workloads in the cluster'
```

Comment:
iiuc this means deleting all workflows? If so please add this info

Reply:
```suggestion
        f'Planning to delete all ({len(workloads)}) workloads in the cluster'
```
Like this?

Reply:
Or just

```suggestion
        f'Planning to delete all workloads in the cluster'
```

?

Reply:
yep